account_multicurrency_revaluation/model/res_company.py

```diff
@@ -28,36 +11,36 @@ class ResCompany(models.Model):
     revaluation_loss_account_id = fields.Many2one(
```

Comment:
A question could arise here. 
Since v9, most of the accounting set up is done in the set up menu.
For me, having the set up in the res.company is fine but does not follow the new trend.

I would suggest to change this set up to the accounting/settings section (nice to have)